packages/model-viewer/src/features/loading.ts

```diff
@@ -469,6 +469,8 @@ export const LoadingMixin = <T extends Constructor<ModelViewerElementBase>>(
             this.dispatchEvent(new CustomEvent('poster-dismissed'));
           });
         }, {once: true});
+      } else {
+        this[$transitioned] = true;
```

Comment:
Found a small bug where when  things  changed too quickly, the transitioned event would not fire and then the state would get messed up. This ensures the state is correct for the poster to dismiss.